oct_levitation:
  control_freq: 100
  sim_mode: true
  publish_computation_time: true
  pose_tracking: true # will require reference pose to be published. Check node for more details.